### DISCUSSION TITLE
Text Classification

### DISCUSSION THREAD

**Discussion by nivedvenugopalan** [explosion/spaCy#9831] [Help: Coding & Implementations] [answered]
I am learning spacy and trying to do text classification. I searched all over youtube and google but could not find a correct way of doing it. Can anyone specify the correct way to do it in spacy3.2??

**Comment by polm** [comment, marked as answer]
Please take a look at the [example projects](https://github.com/explosion/projects/tree/v3/tutorials), several deal with textcat and you can use them as starting points.

**Reply by nivedvenugopalan** [reply]
Thank You!
